packages/uniforms-material/src/RadioField.tsx

```diff
@@ -48,6 +49,7 @@ function Radio({
       onChange={(event: any) => disabled || onChange(event.target.value)}
       ref={inputRef}
       value={value ?? ''}
+      row={row}
```

Comment:
Let's keep these ordered:
```suggestion
      row={row}
      value={value ?? ''}
```